src/common/perf_counters.h

```diff
@@ -280,26 +281,45 @@ class PerfCountersBuilder
   PerfCountersBuilder(CephContext *cct, const std::string &name,
 		    int first, int last);
   ~PerfCountersBuilder();
+
+  // prio values: higher is better, and higher values get included in
+  // 'ceph daemonperf' (and similar) results.
+  //   10 = always, critical stat, very interesting
+  //   8 = quite interesting
+  //   5 = somewhat interesting
```

Comment:
It would be nice to have some named constants for these levels, for use in the declarations of perfcounters in the code (and to implicitly encourage people to use the levels for their intended meanings)